### ISSUE TITLE
Keywords underneath running head instead of abstract

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface#433]
Below the running head of articles which contain a keywords tag (artkwds), it's showing the word keywords: with no data, even though there is data in the XML.  Here's the search set and an example:

https://stage.pep-web.rocks/search/document/AJP.067.0097A?page=P0097&q=art_kwds%3A*

![image](https://user-images.githubusercontent.com/4837554/114636704-aa878380-9c95-11eb-8627-2916fd84ccd8.png)

it is useful to display keywords right below the abstract, which is where they typically appear in print.

So far that's the only place I've found them...here's a sample of a few journals.  Of course, the secondary problem in the formatting of the results is there's actually no keywords shown.)

In AJP
![image](https://user-images.githubusercontent.com/4837554/114636670-95aaf000-9c95-11eb-9a21-f82207efc085.png)

In IJP
![image](https://user-images.githubusercontent.com/4837554/114637018-6052d200-9c96-11eb-9994-4f089c478298.png)

In PPSY
![image](https://user-images.githubusercontent.com/4837554/114637113-92643400-9c96-11eb-8155-31db7584eece.png)

**Comment by bakerac4** [comment]
@nrshapiro here are keywords from a working article

```
<artkwds><impx type="KEYWORD">Henrik Ibsen</impx><impx type="KEYWORD">Peer Gynt</impx><impx type="KEYWORD">tears</impx><impx type="KEYWORD">reparation</impx><impx type="KEYWORD">mourning</impx><impx type="KEYWORD">Melanie Klein</impx></artkwds>
```

and here are keywords from this article

```
<artkwds>hope; trauma; despair.</artkwds>
```

Ideally we wouldn't need to handle both use cases and the API would standardize on the first. Thoughts?

**Comment by bakerac4** [comment]
@nrshapiro Just wanted to ping you again in case you missed the notification for this

**Comment by nrshapiro** [comment]
@abaker-gavant 

Can you please supply the example article IDs?

**Comment by nrshapiro** [comment]
@abaker-gavant 

Actually, I found the first one...and it meets my suspicions.  Actually, the only articles like got the impx tagging are the ones from journals from Taylor and Francis.  So it's not the , vs ; which is what I thought you were talking about.

The good news is I've added code to the processing software to add in the impx tags on the keywords.
But there are a lot of files to reprocess.  It won't be finished until tomorrow, and it takes 4 hours or so to load it to AWS.  So won't be online until Wed.

Of course, meanwhile, don't forget the main issue here is WHERE the keywords appear.
